src/main/deploy/auton/BlueAmpWing1Center1.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE auton SYSTEM "auton.dtd">
+<auton>
+
+<primitive id= "RESET_POSITION_PATH_PLANNER"
+        pathname= "BlueAmpWing1Center1"/>
+
+<snippet file = "ShootWhenStop.xml"/>
+
+<primitive id = "DRIVE_PATH_PLANNER"
+        pathname = "BlueAmpWing1Center1">
+        <zone filename = "BlueIntakeWingNote1.xml"/>
```

Comment:
These both run the same path, so instead of a snippet in the middle, you need to put the zone in for the launch